proto/shentu/bounty/v1/genesis.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+package shentu.bounty.v1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos_proto/cosmos.proto";
+import "shentu/bounty/v1/bounty.proto";
+
+option go_package = "github.com/shentufoundation/shentu/x/bounty/types";
+
+message GenesisState {
+  option (gogoproto.equal) = false;
+  option (gogoproto.goproto_getters) = false;
+
+  repeated Program programs = 1 [ (gogoproto.moretags) = "yaml:\"programs\"", (gogoproto.nullable) = false ];
+}
```

Comment:
I think a `program_id` should also be added here, similar to gov. It will be used in import.

Reply:
`Program` contains `program_id`